src/content/contributing/translation-program/resources/index.md

```diff
@@ -44,7 +44,7 @@ To keep up-to-date with the latest Translation Program progress, you can follow
 
 ## Office hours for translators {#office-hours}
 
-We have office hours for translators on the second Wednesday of every month. These are held in the #office-hours voice channel on the [ethereum.org Discord](https://discord.gg/geKhWjtF), where you can also find the exact times and additional details.
+We have office hours for translators on the second Wednesday of every month. These are held in the #office-hours voice channel on the [ethereum.org Discord](https://discord.gg/Sfz7jqpBWU), where you can also find the exact times and additional details.
```

Comment:
```suggestion
We have office hours for translators on the second Wednesday of every month. These are held in the #office-hours voice channel on the [ethereum.org Discord](/discord/), where you can also find the exact times and additional details.
```

I think given the text here, we can just leave this as a link to the Discord. However, agree we need a working link, so updating to the discord path of the site which will always work.